Parsers/ASimAuditEvent/Parsers/ASimAuditEventVMwareCarbonBlackCloud.yaml

```diff
@@ -0,0 +1,186 @@
+Parser:
+  Title: Audit Event ASIM parser for VMware Carbon Black Cloud
+  Version: '0.1.0'
+  LastUpdated: Oct 23 2023
+Product:
+  Name: VMware Carbon Black Cloud
+Normalization:
+  Schema: AuditEvent
+  Version: '0.1'
+References:
+- Title: ASIM Audit Event Schema
+  Link: https://aka.ms/ASimAuditEventDoc
+- Title: ASIM
+  Link: https://aka.ms/AboutASIM
+- Title: VMware Carbon Black Cloud documentation
+  Link: 
+    https://developer.carbonblack.com/reference/carbon-black-cloud/data-forwarder/schema/latest/endpoint.event-1.0.0/
+    https://developer.carbonblack.com/reference/cb-threathunter/latest/event-search-fields/
+Description: |
+  This ASIM parser supports normalizing VMware Carbon Black Cloud logs to the ASIM Audit Event normalized schema. VMware Carbon Black Cloud events are captured through VMware Carbon Black Cloud data connector which ingests Carbon Black Audit, Notification and Event data into Microsoft Sentinel through the REST API.
```

Comment:
@jayeshprajapaticrest @niralishah-crest
I understand these are VMWare application Admin audit logs.
Does Carbon Black logs user specific audit events from user's endpoint?

Reply:
@vakohl Carbon Black logs audit events which are performed by the user on Carbon Black Cloud Console.